### PR TITLE
docs: tidy up Clef Clique signing page

### PR DESCRIPTION
**Pull request by jmcook1186**
Refining page ready for inclusion on new website. Mirroring content changes here too.
Mostly stylistic and layout changes.

### CODE REVIEW

File: docs/_clef/CliqueSigning.md [hunk 1]
```diff
@@ -66,16 +77,16 @@ INFO [06-16|11:14:54.125] Persisted trie from memory database      nodes=1 size=
 INFO [06-16|11:14:54.125] Successfully wrote genesis state         database=lightchaindata hash=187412..4deb98
 ```
 
-At this point, we have a Clique network which we can start sealing on.
+At this point a Clique network exists with blocks that can be signed.
```
Comment (holiman) [inline docs/_clef/CliqueSigning.md:80]:
```suggestion
At this point a Clique network exists with blocks that can be signed.
```
Well, we've initiated geth with a geneiss, but no blocks exist yet.

File: docs/_clef/CliqueSigning.md [hunk 2]
```diff
@@ -66,16 +77,16 @@ INFO [06-16|11:14:54.125] Persisted trie from memory database      nodes=1 size=
...
 
-## Part 1: Prepping Clef
+## Prepping Clef
 
-In order to make use of `clef` for signing, we need to do a couple of things.
+In order to make use of Clef for signing:
```
Comment (holiman) [inline docs/_clef/CliqueSigning.md:84]:
```suggestion
In order to make use of `clef` for signing:
```
When we talk about the binary itself, I think we should use `clef`. I may have used "Clef" here and there, but that's not right IMO

**Review by holiman — COMMENTED** [review]
Looks mostly good to me

File: docs/_clef/CliqueSigning.md (hunk 1 above)
Comment (jmcook1186) [reply docs/_clef/CliqueSigning.md:80]: yes ofc - thanks!

```suggestion
At this point a Geth has been initiated with a genesis configuration.
```

File: docs/_clef/CliqueSigning.md (hunk 2 above)
Comment (jmcook1186) [reply docs/_clef/CliqueSigning.md:84]: ok!

File: docs/_clef/CliqueSigning.md [hunk 3]
```diff
@@ -384,10 +415,15 @@ clef --keystore ./ddir/keystore \
   --suppress-bootwarn --signersecret ./clefpw \
     attest  `sha256sum rules.js | cut -f1`
 ```
-The normal startup command for `clef`:
-```bash
+The normal startup command for Clef:
```
Comment (jmcook1186) [inline docs/_clef/CliqueSigning.md:418]:
```suggestion
The normal startup command for `clef`:
```

File: docs/_clef/CliqueSigning.md [hunk 4]
```diff
@@ -66,16 +77,16 @@ INFO [06-16|11:14:54.125] Persisted trie from memory database      nodes=1 size=
...
 
-## Part 1: Prepping Clef
+## Prepping Clef
 
-In order to make use of `clef` for signing, we need to do a couple of things.
+In order to make use of Clef for signing:
 
-1. Make sure that `clef` knows the password for the keystore.
-2. Make sure that `clef` auto-approves clique signing requests.
+1. Ensure Clef knows the password for the keystore.
```
Comment (jmcook1186) [inline docs/_clef/CliqueSigning.md:86]:
```suggestion
1. Ensure `clef` knows the password for the keystore.
```

File: docs/_clef/CliqueSigning.md [hunk 5]
```diff
@@ -66,16 +77,16 @@ INFO [06-16|11:14:54.125] Persisted trie from memory database      nodes=1 size=
...
+2. Ensure Clef auto-approves clique signing requests.
```
Comment (jmcook1186) [inline docs/_clef/CliqueSigning.md:87]:
```suggestion
2. Ensure `clef` auto-approves clique signing requests.
```

File: docs/_clef/CliqueSigning.md [hunk 6]
```diff
@@ -66,16 +77,16 @@ INFO [06-16|11:14:54.125] Persisted trie from memory database      nodes=1 size=
...
 
-These two things are independent of each other. First of all, however, we need to `init` clef (for this test I use the password `clefclefclef`)
+These two things are independent of each other. First of all, however, Clef must be initiated (for this example the password is `clefclefclef`)
```
Comment (jmcook1186) [inline docs/_clef/CliqueSigning.md:89]:
```suggestion
These two things are independent of each other. First of all, however, `clef` must be initiated (for this example the password is `clefclefclef`)
```

File: docs/_clef/CliqueSigning.md [hunk 7]
```diff
@@ -113,33 +124,35 @@ Repeat password:
...
 
-At this point, if we were to use clef as a sealer, we would be forced to manually click Approve for each block, but we would not be required to provide the password.
+At this point, if Clef is used as a sealer, each block would require manual approval, but without needing to provide the password.
 
-#### Testing stored password
+### Testing stored password
 
-Let's test using the stored password when sealing Clique-blocks. Start `clef` with
-```
+To test that the stored password is correct and being properly handled by Clef, first start Clef:
```
Comment (jmcook1186) [inline docs/_clef/CliqueSigning.md:133]:
```suggestion
To test that the stored password is correct and being properly handled by Clef, first start `clef`:
```

File: docs/_clef/CliqueSigning.md [hunk 8]
```diff
@@ -191,21 +205,28 @@ function ApproveSignData(r){
...
+
+```terminal
 Decrypt master seed of clef
 Password: 
 INFO [06-16|13:49:00.298] Ruleset attestation updated              sha256=54aae496c3f0eda063a62c73ee284ca9fae3f43b401da847ef30ea30e85e35d1
 ```
-And then we can start clef, pointing out the `rules.js` file. OBS: if you later modify this file, you need to redo the `attest`-step.
-```
+
+And Clef can be started, pointing out the `rules.js` file.
```
Comment (jmcook1186) [inline docs/_clef/CliqueSigning.md:221]:
```suggestion
And `clef` can be started, pointing out the `rules.js` file. 
```

File: docs/_clef/CliqueSigning.md [hunk 9]
```diff
@@ -191,21 +205,28 @@ function ApproveSignData(r){
...
-```
+
+And Clef can be started, pointing out the `rules.js` file. 
+
+```sh
 $ clef --keystore ./ddir/keystore --configdir ./clef --chainid 15  --suppress-bootwarn  --rules ./rules.js 
 ```
 
-Once `geth` starts asking it to seal blocks, we will now see the data. And from that, we can decide on how to make a rule which allows signing clique headers but nothing else.
+Once Geth starts asking Clef to seal blocks, the data will be displayed. From that data, rules can be defined that allow signing clique headers but nothing else.
```
Comment (jmcook1186) [inline docs/_clef/CliqueSigning.md:227]:
```suggestion
Once Geth starts asking `clef` to seal blocks, the data will be displayed. From that data, rules can be defined that allow signing clique headers but nothing else.
```

File: docs/_clef/CliqueSigning.md [hunk 10]
```diff
@@ -252,7 +273,7 @@ If we wanted our js to be extremely trustless/paranoid, we could (inside the jav
   0000000000000000,
 ]
 ```
-However, we can also use the `messages`. They do not come from the external caller, but are generated from the `clef` internals: `clef` parsed the incoming request and verified the Clique wellformedness of the content. So we let's just check for such a message:
+However, `messages` could also be used. They do not come from the external caller, but are generated from the Clef internals: Clef parsed the incoming request and verified the Clique wellformedness of the content. The following simply checks for such a message:
```
Comment (jmcook1186) [inline docs/_clef/CliqueSigning.md:276]:
```suggestion
However, `messages` could also be used. They do not come from the external caller, but are generated inernally: `clef` parsed the incoming request and verified the Clique wellformedness of the content. The following simply checks for such a message:
```

File: docs/_clef/CliqueSigning.md [hunk 11]
```diff
@@ -273,24 +294,33 @@ function ApproveSignData(r){
...
+
+```terminal
 Decrypt master seed of clef
 Password: 
 INFO [06-16|14:18:53.476] Ruleset attestation updated              sha256=7d5036d22d1cc66599e7050fb1877f4e48b89453678c38eea06e3525996c2379
 ```
-Run clef
-```
-$ clef --keystore ./ddir/keystore --configdir ./clef --chainid 15  --suppress-bootwarn  --rules ./rules.js 
+Run clef:
```
Comment (jmcook1186) [inline docs/_clef/CliqueSigning.md:311]:
```suggestion
Run `clef`:
```

File: docs/_clef/CliqueSigning.md [hunk 12]
```diff
@@ -273,24 +294,33 @@ function ApproveSignData(r){
...
-Run geth
-```
+Run Geth:
+
+```sh
 $ geth  --datadir ./ddir --signer ./clef/clef.ipc --mine
 ```
-And you should now see `clef` happily signing blocks:
-```
+And Clef should now happily sign blocks:
```
Comment (jmcook1186) [inline docs/_clef/CliqueSigning.md:321]:
```suggestion
And `clef` should now happily sign blocks:
```

**Review by holiman — APPROVED** [review]
LGTM, thanks!

**Pull request merged**